src/UI/MainMenu.cpp

```diff
@@ -1,11 +1,10 @@
 #include "MainMenu.h"
-
-#include "Game-core/game.h"
-
-
+#include "SFML/Graphics.hpp"
+#include "Game-Core/game.h"
+#include <iostream>
 
 void game::MainMenu::start( sf::RenderWindow * w ) {
-	gmenu::Menu menu( w );
+	gmenu::Menu men( w );  // remeber to change this back to menu after completion
```

Comment:
I think you missed this. Also out of curiosity, why did you have to make this change?

Reply:
Oh sorry actually in order to fix the error temporary is used menu[] array so i just changed the name there so it wouldn't give me error. You can see menu array is declared at the bottom of MainMenu class in MainMenu.h under the private part.

Reply:
If you don't require anymore. Please remove these workaround code.